core/Tracker/Visit/VisitProperties.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Tracker\Visit;
+
+/**
+ * Holds temporary data for tracking requests.
+ *
+ * RequestProcessors
+ */
+class VisitProperties
```

Comment:
I don't remember if there was another reason, but mostly, calling the information `Visitor` information is incorrect. It's technically information about the last known visit of a visitor (so visit info). For example, the location_country attribute might be for Germany, but a visitor's location may have changed many times. I wanted to create a value object for visits, but `class Visit` was already taken.